install_bes_source_deps.sh

```diff
@@ -2,18 +2,31 @@
 
```

Comment:
Is this file part of the travis-ci build? There's no comments saying what this file is for.